src/lib/components/Select/index.js

```diff
@@ -69,21 +69,25 @@ class Select extends BaseElement {
    * @returns void
    */
   handleKeyDown(e) {
-    if (e.key !== 'ArrowUp' && e.key !== 'ArrowDown') {
-      return;
-    }
-    e.preventDefault();
-    const increment = e.key === 'ArrowUp' ? -1 : 1;
-    const lis = this.querySelectorAll('li');
+    switch (e.key) {
+      case 'Escape':
+        this.toggleState();
+        break;
+      case 'ArrowUp':
+      case 'ArrowDown':
```

Comment:
One other thing I just noticed (difference between this component and a native select) is I can't press the up/down arrows to open a closed select element when it's focused.